frontend/src/pages/Login/Login.jsx

```diff
@@ -37,7 +38,7 @@ export function Login() {
 
   function loginUser(e) {
     e.preventDefault();
-    return fetch("http://localhost:3500/auth/login", {
+    return fetch(AUTH_LOGIN, {
```

Comment:
While calling the apis you can just import the endpoint and do `{END_POINT}/auth/login` to get the complete url for hitting the api